app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/ImportFilesScreen.kt

```diff
@@ -189,39 +171,66 @@ private fun ImportFilesScreen(
         content = {
             Column(
                 modifier = Modifier
-                    .padding(horizontal = Margin.Medium, vertical = Margin.Large)
+                    .widthIn(max = Dimens.MaxSinglePaneScreenWidth)
```

Comment:
I think the max width should go somewhere higher if possible so it's applied to all of the screens of this activity

Reply:
I moved it to StockyBottomButtonScafflod as all screen using it should be constrained like this